### ISSUE TITLE
Can the option to duplicate data be exposed in a better way?

### ISSUE THREAD

**Issue by dtasev** [mantidproject/mantidimaging#554]
The only way to discover this feature right now is to right click the stack and read all the options.

This isn't great as no one ever does that unless by accident. Can there be a better way to expose this feature?

- A new button in operations window - apply to duplicated data?
  - Does that create too many buttons there after #539 ?
- Offer to delete old dataset once the operation is complete

**Comment by dtasev** [comment]
Closing in favour of #578
